Signal/translations/en.lproj/Localizable.strings

```diff
@@ -1534,6 +1534,66 @@
 /* Title for the 'scan QR code' view. */
 "SCAN_QR_CODE_VIEW_TITLE" = "Scan QR Code";
 
+/* Indicates a delay of zero seconds, and that 'screen lock activity' will timeout immediately. */
+"SCREEN_LOCK_ACTIVITY_TIMEOUT_NONE" = "None";
+
+/* Title for alert indicating that screen lock could not be disabled. */
+"SCREEN_LOCK_DISABLE_FAILED" = "Screen Lock Could Not Be Disabled";
+
+/* Title for alert indicating that screen lock could not be enabled. */
+"SCREEN_LOCK_ENABLE_FAILED" = "Screen Lock Could Not Be Enabled";
+
+/* Indicates that an unknown error occurred while using Touch ID or Face ID. */
+"SCREEN_LOCK_ENABLE_UNKNOWN_ERROR" = "Touch ID and Face ID could not be accessed.";
+
+/* Indicates that Touch ID/Face ID authentication failed. */
+"SCREEN_LOCK_ERROR_LOCAL_AUTHENTICATION_FAILED" = "Authentication with Touch ID or Face ID failed.";
+
+/* Indicates that Touch ID/Face ID is 'locked out' on this device due to authentication failures. */
+"SCREEN_LOCK_ERROR_LOCAL_AUTHENTICATION_LOCKOUT" = "Touch ID or Face ID is locked out; too many failures.";
+
+/* Indicates that Touch ID/Face ID are not available on this device. */
+"SCREEN_LOCK_ERROR_LOCAL_AUTHENTICATION_NOT_AVAILABLE" = "Touch ID or Face ID is not available on this device.";
```

Comment:
copy nit: "Neither Touch ID nor Face ID is available on this device." or maybe better "Your device does not support Screen Lock, which requires either Touch ID or Face ID."

But a bigger question - do we want to hard fail when these aren't available? Or just *prefer* them when they are available, and fall back to lock screen PIN entry. 

Personally, I'd prefer the latter.

Reply:
Done - I've tried to honor the spirit of this comment in all of the l10n strings.